app/models/container_image_registry.rb

```diff
@@ -2,13 +2,13 @@ class ContainerImageRegistry < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => "ems_id"
 
   # Associated with images in the registry.
-  has_many :container_images, :dependent => :nullify
+  has_many :container_images # Destroy will be handled by purger
   has_many :containers, :through => :container_images
   has_many :container_groups, :through => :container_images
 
   # Associated with serving the registry itself - for openshift's internal
   # image registry. These will be empty for external registries.
-  has_many :container_services
+  has_many :container_services, :dependent => :destroy
```

Comment:
seeing tens of registries to many hundreds of services.  Should be handled if registries are removed with the container manager.